src/library_pthread.js

```diff
@@ -400,6 +402,32 @@ var LibraryPThread = {
         'workerID': PThread.nextWorkerID++,
 #endif
       });
+      return ready;
+    },
+
+    loadWasmModuleToWorkers: function(onMaybeReady) {
```

Comment:
The name similarity - just adding `s` at the end - seems maybe confusing. Given such similar names I'd expect the first function to operate on a single worker and the second function to operate on them all, but the parameters and the code inside don't seem consistent with that expectation.

This function looks like it sets up a callback for when we are ready, so perhaps `waitForPool` or something similar?

Reply:
FWIW that's how it was called in one of the preamble/postamble already. But yeah, renaming sounds good.

`waitForPool` is not sufficient because it doesn't convey the initialization part - this function actually sends Wasm module to all the workers, not just waits for it.

Maybe `initAllWorkerRuntimes` or something?

Reply:
Renamed to `loadWasmModuleToAllWorkers` in the extracted PR, hope that's a sufficient distinction.